proxies/events-graphql-proxy/src/datasources/LinkedEventsDataSource.ts

```diff
@@ -0,0 +1,6 @@
+import DataSourceWithContext from './DataSourceWithContext';
+
+abstract class LinkedEventsDataSource extends DataSourceWithContext {
+  override baseURL = process.env.GRAPHQL_PROXY_API_BASE_URL;
```

Comment:
Minor thing, but the process.env things could be collected to some app configuration file or constants.

Reply:
This could be done when venue server is incorporated to a server code and common code is created-